docs/griptape-framework/data/extraction-engines.md

```diff
@@ -0,0 +1,94 @@
+# Overview 
+Extraction Engines are used to extract data from text. 
+
+## CSV Extraction Engine
+
+Used to extract data from text using a CSV file. You can set a custom [prompt driver](../../griptape-framework/structures/prompt-drivers.md) or [chunker](../../griptape-framework/data/chunkers.md). 
+You can use it along with [Extraction Tasks](../../griptape-framework/structures/tasks.md)
+
+```python
+from griptape.engines import CsvExtractionEngine
+from griptape.artifacts import ListArtifact, ErrorArtifact
+from griptape.drivers import OpenAiChatPromptDriver
+
+# Initialize the CsvExtractionEngine instance
+csv_engine = CsvExtractionEngine(prompt_driver=OpenAiChatPromptDriver(model="gpt-4", temperature=0.3))
```

Comment:
What's the reasoning for explicitly providing a Prompt Driver? Does this only work with this temperature?

Reply:
There was no reason to use the prompt driver, just added it to the example so people know that a prompt driver can be used. The temperature was totally random. I can remove the prompt driver from the example if you want.